### ISSUE TITLE
Enhancement: Open Last Workfile setting allow enable/disable per user per application

### ISSUE THREAD

**Issue by BigRoy** [ynput/OpenPype#5697]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Please describe the feature you have in mind and explain what the current shortcomings are?

The opening of the last workfile has been a re-occurring debate on our team. Some like it, some hate it (literally pure hatred, they are steaming red head angry due to it).

Would it make sense to allow this to be an artist specific choice? With AYON this should become quite feasible since we now login per user?
And would it also be possible to allow the user to toggle it per host? Some users said "I only like it for application X; not for Y."

### How would you imagine the implementation of the feature?

The User has user-specific settings that they can enable/override specific to them of which this would be one. If not overridden it could then still use the studio default behavior.

This could live alongside the override per task setting somehow?

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the enhancement request.

### Describe alternatives you've considered:

Let the team fight to the death on the topic. The remaining ones that are alive at the end will decide whether the feature will be enabled or disabled for the rest of eternity.

**Comment by antirotor** [comment]
We have similar requests. Like to be able to invert the logic of the "skip last opened workfile" with some filter - use case is Houdini scene where you want to switch update to manual before opening some heavy scene.

**Comment by mkolar** [comment]
Yes this should be possible to implement per artist in AYON. Yey.
